src/lua/compat.lua

```diff
@@ -58,6 +70,12 @@ local options = {
         brief = YAML_PRETTY_MULTILINE_BRIEF,
         action = internal.yaml_pretty_multiline_toggle,
     },
+    fiber_channel_graceful_close = {
+        default = 'old',
+        obsolete = nil,
+        brief = FIBER_CHANNEL_GRACEFUL_CLOSE_BRIEF,
+        action = internal.fiber_channel_graceful_close_toggle,
+    },
```

Comment:
```lua
compat.fiber_channel_graceful_close = 'old'
compat.fiber_channel_graceful_close = 'new'
```

It sounds a bit artificial. How about another name?

```lua
compat.fiber_channel_close_mode = 'old'
compat.fiber_channel_close_mode = 'new'
```

Reply:
Also looks better to me. Fixed.